test/ParameterStoreBuildWrapperPluginTest.groovy

```diff
@@ -84,5 +125,38 @@ class ParameterStoreBuildWrapperPluginTest {
             assertEquals(ParameterStoreBuildWrapperPlugin.class, result)
         }
     }
+
+    class withGlobalParameter {
+        @After
+        public void reset() {
+            ParameterStoreBuildWrapperPlugin.reset()
+        }
+
+        @Test
+        void addGlobalParameter() {
+            def result = ParameterStoreBuildWrapperPlugin.withGlobalParameter('/path/', [])
+
+            assertEquals([[path: '/path/']], result.globalParameterOptions)
+        }
+
+        @Test
+        void addGlobalParameterWithOptions() {
+            def result = ParameterStoreBuildWrapperPlugin.withGlobalParameter('/path/', [recursive: true, basename: 'relative'])
+
+            assertEquals([[path: '/path/', recursive: true, basename: 'relative']], result.globalParameterOptions)
+        }
+
+        @Test
+        void addMulitpleGlobalParameters() {
+            ArrayList expected = []
+            def result = ParameterStoreBuildWrapperPlugin.withGlobalParameter('/path/', [])
+                                                         .withGlobalParameter('/path2/', [recursive: true])
+                                                         .withGlobalParameter('/path3/', [basename: 'something'])
+            expected << [path:'/path/']
+            expected << [path: '/path2/', recursive: true]
+            expected << [path: '/path3/', basename: 'something']
+            assertEquals(expected, result.globalParameterOptions)
+        }
+    }
```

Comment:
I like this.  You're showing that you can add more than  one global parameter, and that  they don't overwrite one another.